src/test/java/org/cactoos/iterator/NoNullsTest.java

```diff
@@ -48,27 +49,39 @@
 
     @Test
     public void nextThrowsErrorIfNull() {
-        this.exception.expect(IllegalStateException.class);
-        this.exception.expectMessage(
-            new StringContains(
-                "Item #0 of org.cactoos.iterator.NoNullsTest"
-            )
-        );
-        new NoNulls<>(
-            new Iterator<Integer>() {
-                @Override
-                public boolean hasNext() {
-                    return true;
-                }
+        MatcherAssert.assertThat(
+            "Must throw exception",
+            () -> new NoNulls<>(
+                new Iterator<Integer>() {
+                    @Override
+                    public boolean hasNext() {
+                        return true;
+                    }
 
-                @Override
-                public Integer next() {
-                    return null;
+                    @Override
+                    public Integer next() {
+                        return null;
+                    }
+
+                    @Override
+                    public String toString() {
+                        return "Iterator@NoNullsTest";
+                    }
                 }
-            }
-        ).next();
+            ).next(),
+            new Throws<>(
+                "Item #0 of Iterator@NoNullsTest is NULL",
+                IllegalStateException.class
+            )
+        );
     }
 
+    /*
+    * @todo #1039:15min Currently it's impossible to match error messages
+    *  by a pattern or partially. Replace {@link Rule} with {@link Throws} after
+    *  <a href="https://github.com/llorllale/cactoos-matchers/issues/108">llorllale/cactoos-matchers#108</a>
```

Comment:
@atapin it would also make sense to open a ticket so that `toString` is implemented on `Immutable`. You should do it and get some more points :)

Reply:
@victornoel thanks, good point! The issue is here #1102